Nim/Program.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+
+namespace NimGame
+{
+    internal class Program
+    {
+        public static void Main(string[] args)
+        {
+            int matchesNumber = 24;
+            int aiChoice = 3;
+            Console.WriteLine("Welcome to Nim!");
+
+            while (matchesNumber > 0)
+            {
+                Console.WriteLine("\nCurrent matches: " + matchesNumber);
+                Console.WriteLine("How many matches do you want to draw?");
+                string matchesText = Console.ReadLine();
+
+                if (int.TryParse(matchesText, out int playerChoice) && playerChoice > 0 && playerChoice <= 3)
```

Comment:
But also, this can be simplified by inverting the if:
```csharp
if(!input.Tryparse...){
   Console.WriteLine("Error");
   continue;
}

// the other code which was the `true` case before here.
```